### PR TITLE
Add support for VLP-32

### PR DESCRIPTION
**Pull request by skoushan**
Differences between VLP-32 and VLP-16 accounted for:
- new Data Packet structure
- new laser calibration file

VeloView provided running on my Mac:
2017-08-29-16-31-38_velodyne-hdl-32-data
2017-08-29-16-32-36_velodyne-hdl-32-data

Rviz running new driver:
rviz_screenshot_2017_08_29-16_27_34
rviz_screenshot_2017_08_29-16_27_59

I noticed that brain points to the `accumulate_slices` branch, which is why I based this off of that.

### CODE REVIEW

**Comment by skoushan** [comment]
Looking into testing...

**Comment by skoushan** [comment]
Looks like tests have been broken since a couple commits back, so we can ignore them them for now

File: velodyne_driver/src/driver/driver.cc [hunk 1]
```diff
@@ -58,6 +58,11 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
       packet_rate = 754;             // 754 Packets/Second for Last or Strongest mode 1508 for dual (VLP-16 User Manual)
       model_full_name = "VLP-16";
     }
+  else if (config_.model == "VLP32")
```
Comment (brandonmoak) [inline velodyne_driver/src/driver/driver.cc:61]: we should call the device `VLP32C`
there will be other velodyne devices that have 32 lasers that will have different density patterns

File: velodyne_pointcloud/launch/VLP32_points.launch [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!-- -*- mode: XML -*- -->
+<!-- run velodyne_pointcloud/CloudNodelet in a nodelet manager for a VLP-16 -->
+
+<launch>
+
+  <!-- declare arguments with default values -->
+  <arg name="calibration" default="$(find velodyne_pointcloud)/params/VLP32.yaml"/>
```
Comment (brandonmoak) [inline velodyne_pointcloud/launch/VLP32_points.launch]: make 32C

File: velodyne_driver/src/driver/driver.cc (hunk 1 above)
Comment (brandonmoak) [reply velodyne_driver/src/driver/driver.cc:61]: actually, leave this as is

**Comment by brandonmoak** [comment]
:+1:

**Review by brandonmoak — APPROVED** [review]

**Review by brandonmoak — CHANGES_REQUESTED** [review]
Look into the firing sequence again.

If lasers are fired 2us apart, then it doesn't fit into a 55us block period

**Comment by skoushan** [comment]
Made the fix

**Review by brandonmoak — APPROVED** [review]

**Pull request merged**